File: JavaScript/ReverseString.js
s = ["h","e","l","l","o"]

var reverseString = function(s) {
    for(s1 = 0, s2 = s.length -1; s1 < s2; s1++, s2--){
           var temp = s[s1] 
           s[s1] = s[s2] 
           s[s2] = temp 
       }
        console.log(s) 
};

reverseString(s)

File: Python/ReverseString.py
s = ["h", "e", "l", "l", "o"]

def reverseString():
    start = 0
    end = len(s) - 1
    while start < end:
        s[start], s[end] = s[end], s[start]
        start += 1
        end -= 1

reverseString()
print(s)